jubilant/_juju.py

```diff
@@ -695,3 +695,12 @@ def _format_config(k: str, v: ConfigValue) -> str:
     if isinstance(v, bool):
         v = 'true' if v else 'false'
     return f'{k}={v}'
+
+
+def _status_diff(old: Status | None, new: Status) -> str:
+    if old is None:
+        return '\n'.join('+ ' + x for x in _pretty.gron(new))
```

Comment:
```suggestion
        return '\n'.join(f'+ {x}' for x in _pretty.gron(new) if not x.startswith('.controller.timestamp'))
```

Or maybe just set `old_lines` to an empty list and let the differ do the formatting.

Reply:
Good idea - done.